README.md

```diff
@@ -66,8 +67,6 @@ The Open Quantum Safe project is led by [Douglas Stebila](https://www.douglas.st
 
 ## Acknowledgments
 
-Financial support for the development of Open Quantum Safe has been provided by Amazon Web Services and the Canadian Centre for Cyber Security.
-
-We'd like to make a special acknowledgement to the companies who have dedicated programmer time to contribute source code to OQS, including Amazon Web Services, evolutionQ, Microsoft Research, Cisco Systems, IBM Research and Fachhochschule Nordwestschweiz.
+Most effort in this project has been provided by individual contributors working in their own time and out of personal interest to see how PQ crypto integrates into existing software stacks.
```

Comment:
If you would like the README to include a statement acknowledging individual contributors, that's fine. However this statement goes beyond that in ways that I do not think should appear in the README.

Reply:
In which way? How would you rephrase?

Reply:
How about:
```suggestion
This project has benefited substantially from the efforts of individual volunteer contributors.
```

Reply:
This text proposal is a true statement about practically any OSS project -- and it doesn't mention the "full software stack" aspect that makes/made this work particularly challenging; your wording could equally well be applied to all OQS sub projects or be left away entirely.

And for the avoidance of doubt: I do not want to stress the aspect of volunteerism (I presume meaning people working for free/outside of employment) but of individuals going beyond their assigned tasks; this for example also includes people like @xvzcf and practically all external contributors -- of whom I don't know their employment status -- but do know about (and wanted to acknowledge with my proposal) their personal interest and commitment.

Reply:
```suggestion
This project has benefited substantially from the efforts of individual contributors motivated to see how PQ crypto integrates into existing software stacks.
```

Reply:
I still see a marked difference between the wording "benefited substantially from" and "most effort has been provided" (by individual contributors): Here's the count of PRs done by "individual contributors" and contributors doing this as part of their employment:

145 to 36 contributions -- provided by 16 individuals to 4 employed ones. Thus, in both counts about 80% has been done by individual contributors, in my eyes justifying the wording I proposed.

Reply:
Resolve this as you see fit.

Reply:
Done